src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int STACK_CAPACITY = 12;
+    private static final int EMPTY_STACK_VALUE = -1;
+    private static final int STACK_ISFULL = 11;

```

Comment:
This constant is related to STACK_CAPACITY . Changing only one leads to an error (inconsistency).